File: 0034-find-first-and-last-position-of-element-in-sorted-array/0034-find-first-and-last-position-of-element-in-sorted-array.cpp
class Solution {
public:
    vector<int> searchRange(vector<int>& nums, int target) {
        int s=-1,e=-1;
       for(int i=0;i<nums.size();i++)
       {
           if(nums[i]==target)
           {
                s=i;
                break;
           }
       }
       for(int i=nums.size()-1;i>=0;i--)
       {
           if(nums[i]==target)
           {
               e=i;
               break;
           }
       }
       return {s,e};
    }
};

File: 0034-find-first-and-last-position-of-element-in-sorted-array/0034-find-first-and-last-position-of-element-in-sorted-array.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var searchRange = function(nums, target) {
    let number=[-1,-1];
    for(let i=0;i<nums.length;i++)
        {
            if(nums[i]==target){
                number[0]=i;
                break;
            }
        }
    for(let i=nums.length-1;i>=0;i--)
        {
            if(nums[i]==target){
                number[1]=i;
                break;
            }
        }
    
    return number;
};